testing/test_pdb.py

```diff
@@ -583,25 +583,28 @@ def test_parseline_with_rc_commands(tmpdir, monkeypatch):
     monkeypatch.delenv("USERPROFILE", raising=False)
 
     with tmpdir.as_cwd():
-        open(".pdbrc", "w").writelines([
-            "alias myp print(%1)\n",
-        ])
+        with open(".pdbrc", "w") as f:
+            f.writelines([
+                "p 'readrc'\n",
+                "alias myalias print(%1)\n",
+            ])
```

Comment:
Just for reference: pypy needs the file to be closed, and would otherwise read it as empty..